modules/openbsd/openbsd-driver.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018 Balabit
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#ifndef OPENBSD_SYSTEM_DRIVER_H_INCLUDED
+#define OPENBSD_SYSTEM_DRIVER_H_INCLUDED
+
+#include "driver.h"
+#include "logsource.h"
+
+struct OpenBSDDriver;
```

Comment:
We usually use `typedef struct _OpenBSDDriver OpenBSDDriver` instead of forward declaration.
Although is it needed here at all ? I do not see a usage. Possibly this should be moved to the C file.

Reply:
This was unused. I removed it. In the openbsd-driver.c I already use typedef.